telegraf/manifests/deis-monitor-telegraf-daemon.yaml

```diff
@@ -36,18 +38,19 @@ spec:
           - name: "ENABLE_PROMETHEUS"
             value: "true"
           - name: "NSQ_CONSUMER_SERVER"
-            value: $(DEIS_NSQD_SERVICE_HOST):$(DEIS_NSQD_SERVICE_PORT_TRANSPORT)
+            value: "$(DEIS_NSQD_SERVICE_HOST):$(DEIS_NSQD_SERVICE_PORT_TRANSPORT)"
           - name: "NSQ_CONSUMER_TOPIC"
             value: "metrics"
           - name: "NSQ_ENDPOINTS"
-            value: http://$(DEIS_NSQD_SERVICE_HOST):$(DEIS_NSQD_SERVICE_PORT_HTTP)
+            value: "http://$(DEIS_NSQD_SERVICE_HOST):$(DEIS_NSQD_SERVICE_PORT_HTTP)" {{- if env "LOGGER_REDIS_LOCATION" | default .logger_redis_location | ne "off-cluster" }}
```

Comment:
you can put the `if` on the next line and the `{{-` will remove that newline. might be easier to read